tools/gulptasks/dashboards/dist-examples.js

```diff
@@ -38,103 +57,98 @@ async function transformExampleDir(examplesDir) {
     ];
 
     for (const fileName of files) {
-        let filePath = join(examplesDir, fileName);
-        const contents = await readFile(filePath, 'utf-8');
+        let filePath = path.join(examplesDir, fileName);
+        const contents = await fs.readFile(filePath, 'utf-8');
 
         let newContent = contents
             .replaceAll('https://code.highcharts.com/dashboards/', '../../code/')
-            .replaceAll(/(?<!\.src)\.js(?!on)/gmu, '.src.js"');
+            .replaceAll(/(?<!\.src)\.js(?!on)/gmu, '.src.js');
 
         if (fileName === 'demo.html') {
             filePath = filePath.replace('demo.html', 'index.html');
-            newContent = `<link rel="stylesheet" type="text/css" href="./demo.css">
-${newContent}
-<script src="demo.js"></script>
-`;
+            newContent = [
+                '<link rel="stylesheet" type="text/css" href="./demo.css">',
+                newContent,
+                '<script src="demo.js"></script>'
```

Comment:
For CSS we have the relative path, but not for demo.js which is in the same dir.

IMO we should unify both